docker/docker-python/build_python_dockerfile.sh

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env bash
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+set -e
+
+# Check Params
+programname=$0
+
+function usage {
+    echo "usage: $programname [version] [path]"
+    echo "  [version]  Mxnet Version to build"
+    echo "  [path]     Path to MXNet repository (to run tests)"
+    echo " "
+    exit 1
+}
+
+if [ $# -le 1 ]
```

Comment:
Please check both -le and -ge boundary conditions to print usage. 
Code suggestion: 
if [ $# -le 1 ] || [ $# -ge 3 ]
then
    usage
    exit 1
fi